src/BotConfig.ts

```diff
@@ -68,8 +68,7 @@ export default class BotConfig {
 		if ( !settings.request_channels ) throw 'Request channels are not set';
 		this.requestChannels = settings.request_channels;
 
-		//no check, beacause it's a boolean.
-		this.ticketUrlsCauseEmbed = settings.ticketUrlsCauseEmbed;
```

Comment:
Was this done by mistake or does the double negation do some fucky stuff in js / ts?